feconf.py

```diff
@@ -693,10 +693,12 @@ def get_empty_ratings():
 DEFAULT_TOPIC_SIMILARITY = 0.5
 SAME_TOPIC_SIMILARITY = 1.0
 
+# While adding another language, please ensure that the
+# languages are in alphabetical order
```

Comment:
Just to check, would it be better to define this as a list instead? This would depend on which parts of the frontend/backend (excluding tests) make use of this variable -- you probably want to keep it as a dict if there are lots of accesses by key, for example. But if it's just used once and it's going to be sorted each time, then maybe it's better to predefine the order here and just serve it as-is.